solidity-frontend/src/main/java/org/sonarsource/solidity/frontend/SolidityAnalyzer.java

```diff
@@ -0,0 +1,5 @@
+package org.sonarsource.solidity.frontend;
+
+public class SolidityAnalyzer {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this empty class, write its code or make it an "interface". ([squid:S2094](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS2094))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=24&issues=AWRLsd4xfpYBZNsoTGs9&open=AWRLsd4xfpYBZNsoTGs9)

[ISSUE_KEY]: AWRLsd4xfpYBZNsoTGs9